resources/chefdk/msi/source.wxs.erb

```diff
@@ -12,24 +12,57 @@
           Version="$(var.VersionNumber)" Manufacturer="!(loc.ManufacturerName)" UpgradeCode="$(var.UpgradeCode)">
 
     <!--
-      Define the minimum supported installer version (2.0).
-      The install should be done for the whole machine not just the current user
+      Minimum installer version (5.0) - Window 7/Server 2008 R2 and above.
+      The install scope is per machine, not the current user
     -->
-    <Package InstallerVersion="200" InstallPrivileges="elevated"
+    <Package InstallerVersion="500" InstallPrivileges="elevated"
```

Comment:
Is this because the fast MSI stuff requires it, or because this is all we support on ChefDK or some other reason?